sotodlib/tod_ops/jumps.py

```diff
@@ -254,7 +231,7 @@ def _diff_buffed(
     win_size: int,
     make_step: bool,
 ) -> NDArray[np.floating]:
-    win_size = int(win_size)
+    win_size = int(win_size + win_size % 2)
```

Comment:
If win_size can be a float, then `int(win_size + win_size % 2)` is not guaranteed to be even (e.g. win_size = 1.6).